src/tlo/methods/healthsystem.py

```diff
@@ -2738,6 +2748,14 @@ def write_to_log_and_reset_counters(self):
             },
         )
 
+        logger_summary.info(
+            key="Equipment",
+            description="Sets of used equipment for each facility level in this calendar year.",
+            data={
+                "Equipment_By_Level": self._equip_by_level,
```

Comment:
As sets aren't ordered, but we want to the results of two runs of the model to be identical (if same random seed and same parameters), I propose that we use `sorted` that will turn the `set` into `list` that is sorted. 


```suggestion
                "Equipment_By_Level": sorted(self._equip_by_level),
```